### PR TITLE
add support for variables

### PR DESCRIPTION
**Pull request by jasonLaster**
It's possible to include URLs in css variables...

```css
.light-theme {
  --theme-icon-filter: url(chrome://devtools/skin/images/filters.svg#icon-normal-dark);
  --theme-icon-selected-filter: url(chrome://devtools/skin/images/filters.svg#icon-selected-dark);
  --theme-icon-checked-filter: url(chrome://devtools/skin/images/filters.svg#icon-checked-dark);
}
```

This is an initial failing test, I hope to add a fix to index.js soon, but wanted to share early

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -7,25 +7,31 @@ var tests = [{
...
-	options: {} 
+	options: {}
```
Comment (jasonLaster) [inline test.js]: sorry for the whitespace :/

**Comment by jasonLaster** [comment]
This is a simple fix, there might be a more official way to handle it, but... it does the job :)

File: index.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ module.exports = require('postcss').plugin('postcss-url-mapper', function(map, o
 			decl.value = replacer(decl.value, decl.prop);
 		});
 
+		css.walkDecls(/^(--)/, function(decl) {
```
Comment (igoradamenko) [inline index.js]: Hi Jason! Thank you for your PR.

Could you please merge this regex with one's above? I think this has to work:

```js
css.walkDecls(/^(?=--|cue|play|background|content|src|cursor|list-style)/, function(decl) {
  decl.value = replacer(decl.value, decl.prop);
});
```

Comment (igoradamenko) [reply index.js]: It's just because one pass is better than two :-)

Comment (igoradamenko) [reply index.js]: Ah, never mind, I've already done it.

**Pull request merged**